File: programme/maximum-of-numbers/MaximumOfNumbers.java
// Write a programme to print the maximum of given numbers
import java.util.*;
import java.util.Scanner;

public class MaximumOfNumbers {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter numbers : ");
    String input = sc.nextLine();

    String[] inputArr = input.split(" ");
    Integer[] numbers = new Integer[inputArr.length];
    for (int i = 0; i < numbers.length; i++) {
      numbers[i] = Integer.parseInt(inputArr[i]);
    }
    List<Integer> list = Arrays.asList(numbers);

    System.out.print("\nMaximum Number : ");
    list
      .stream()
      .reduce((x, y) -> x > y ? x : y)
      .ifPresent(System.out::println);
    sc.close();
  }
}

File: programme/maximum-of-numbers/maximum-of-numbers.c
// Write a program to print the maximum of given numbers
#include <stdio.h>
#include <limits.h>

int main()
{
    // Initializes the max variable as the minimum integer possible
    int max = INT_MIN, n, test;
    printf("Enter amount of numbers : ");
    scanf("%i", &n);
    printf("\n");

    // Input one number at a time
    while (n > 0)
    {
        printf("Enter number : ");
        scanf("%i", &test);
        if (test > max)
            max = test;
        n--;
    };
    // Output
    printf("\nThe maximum is : %i\n", max);
    return 0;
}
